docs_outdoornav_user_manual/integration_requirements/installation_instructions.mdx

```diff
@@ -0,0 +1,133 @@
+---
+title: "Installation Instructions"
+sidebar_label: "Installation Instructions"
+sidebar_position: 4
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+Prior to installing the OutdoorNav software, ensure that you have completed the relevant [Hardware Kit Checklist](/docs_outdoornav_user_manual/integration_requirements/hardware_integration_requirements/hardware_integration_requirements). If installing on a secondary/backpack/Starter Kit computer, ensure to have also completed the:
+- [Platform Computer Checklist](/docs_outdoornav_user_manual/integration_requirements/interface_control_requirements)
+- [Interface Control Checklist](/docs_outdoornav_user_manual/integration_requirements/interface_control_requirements)
+
+All the following instructions, unless otherwise specified, are to be run on the OutdoorNav computer.
+
+### Install Docker {#install-docker}
+
+``` bash
+cd ~/cpr_outdoornav_launch/scripts/
+./install_docker.sh
+```
+
+### Configure OutdoorNav Sensors {#configure-outdoornav-sensors}
+
+Prior to configuring the sensors, ensure that you have measured the position (XYZ) and orientation (RPY) of each of your sensors w.r.t. the base_link. See your results from the [Integration Requirements](/docs_outdoornav_user_manual/integration_requirements/hardware_integration_requirements).

```

Comment:
1. Do we use "w.r.t." elsewhere? If not, should we spell it out?
2. `base_link`

Reply:
how do you do this syntax in mdx files?

Reply:
haha i guess that was it since it did what I was expecting in the comment.